### DISCUSSION TITLE
Nuget restore for Maui Android and IOS project error message

### DISCUSSION THREAD

**Discussion by snelsn04** [LuckyPennySoftware/MediatR#964] [Q&A]
Getting the following error trying to restore nuget packages for a Maui Android and IOS project. 

 - task: NuGetCommand@2
    displayName: 'NuGet Restore'
    inputs:
      command: 'restore'
      restoreSolution: '$(Build.Repository.LocalPath)/Source/Mobile/Mobile/Mobile.sln'
      feedsToUse: 'config'
      nugetConfigPath: '$(Build.Repository.LocalPath)\Build\NuGet.config'

##[error]The nuget command failed with exit code(1) and error(NU1202: Package MediatR.Extensions.Microsoft.DependencyInjection 11.0.0 is not compatible with net70 (.NETFramework,Version=v7.0). Package MediatR.Extensions.Microsoft.DependencyInjection 11.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
NU1202: Package MediatR.Extensions.Microsoft.DependencyInjection 11.0.0 is not compatible with net70-android33.0 (.NETFramework,Version=v7.0,Profile=android33.0). Package MediatR.Extensions.Microsoft.DependencyInjection 11.0.0 supports: netstandard2.1 (.NETStandard,Version=v2.1)
NU1202: Package MonkeyCache.LiteDB 2.0.1 is not compatible with net70-android33.0 (.NETFramework,Version=v7.0,Profile=android33.0). Package MonkeyCache.LiteDB 2.0.1 supports: net60 (.NETFramework,Version=v6.0)
NU1202: Package Xamarin.Android.Glide 4.13.2.2 is not compatible with net70-android33.0 (.NETFramework,Version=v7.0,Profile=android33.0). Package Xamarin.Android.Glide 4.13.2.2 supports:
  - monoandroid12.0 (MonoAndroid,Version=v12.0)

**Comment by jbogard** [comment]
No idea, I don't do mobile development.
